pypeit/wavecalib.py

```diff
@@ -529,6 +529,9 @@ def build_waveimg(spectrograph, tilts, slits, wv_calib, spat_flexure=None):
     """
     Main algorithm to build the wavelength image
 
+    Only applied to good slits, which means any non-flagged or flagged
```

Comment:
Is it possible to add an `initial=False` as a parameter, so the initial slit edges can be used?

Reply:
yes, but I prefer not to do this in this PR.